notNeededPackages.json

```diff
@@ -839,6 +839,12 @@
             "typingsPackageName": "zone.js",
             "sourceRepoURL": "https://github.com/angular/zone.js",
             "asOfVersion": "0.5.12"
+        },
+        {
+            "libraryName": "mirror",
```

Comment:
Why `mirror` lib should be removed? it seems it don't contain types: https://github.com/mirrorjs/mirror/blob/master/package.json

Reply:
Ok， I will remove it.

Reply:
@plantain-00  Thanks, I have removed it from `notNeededPackages.json`. Please check again.